src/Broker/Broker.h

```diff
@@ -55,8 +55,10 @@
 /** @brief Initialize Broker
  *
  * Broker shall be initialized before starting system's threads.
+ * 
+ * @param[in]    valid_buses   An array of used buses in current broker.
```

Comment:
An array of busses used in the broker instance

Reply:
done